docs/running from source.md

```diff
@@ -69,6 +69,20 @@ It should be dropped as "SNI" into the root folder of the project. Alternatively
 host.yaml at your SNI folder.
 
 
+## Optional: Git
+
+[Git](https://git-scm.com) is required to install some of the packages that Archipelago depends on.
+It may be possible to run Archipelago from source without it, at your own risk.
+
+It is also generally recommended to have Git installed and understand how to use it, especially if you're thinking about contributing.
+Git will allow you to keep your copy of the repository updated without having to redownload the .zip manually.
+It lets you manage branches or forks of the repository with ease, and it will be all but required to submit your changes via a pull request.
```

Comment:
I'm not sure about this section and would personally just drop it. It's not in-depth enough for someone to understand what any of that means, but describing what git is or does would go beyond the scope of this doc. (Also for some people, especially for documentation, the pencil tool on github might be better suited.)
```suggestion
```